### ISSUE TITLE
[DOCS] Top level `plaintext` config does not work in 4.0.0

### ISSUE THREAD

**Issue by v1k45** [maizzle/framework#707]
I upgraded from v3 to v4, the top level `plaintext: true` does not generate plain text templates anymore.

I had to put the plaintext object in `build.templates` to make it work. The documentation still points to a top-level `plaintext` config.

Before:

```js
module.exports = {
  plaintext: true,
  build: {
    templates: {
      destination: {
        path: '../backend/templates/maizzle',
      },
    },
  },
  inlineCSS: true,
  removeUnusedCSS: true,
}

```

After:
```js
module.exports = {
  build: {
    templates: {
      destination: {
        path: '../backend/templates/maizzle',
      },
      plaintext: {
        destination: {
          path: '../backend/templates/maizzle',
        },
      },
    },
  },
  inlineCSS: true,
  removeUnusedCSS: true
}

```

- Maizzle Version: 4.0.0
- Node.js Version: 14.19.3

**Comment by chris74er** [comment]
I still have the problem with plaintext in Front Matter. Is it already fixed?

**Comment by cossssmin** [comment]
Fixed in 4.2.0:

https://github.com/maizzle/framework/releases/tag/v4.2.0
